prettierd_formatter.py

```diff
@@ -29,7 +29,9 @@ def format_with_prettierd(view_or_window, content, file_path):
     cmd = [prettierd_path, "--stdin-filepath", file_path]
     
     try:
-        process = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=os.path.dirname(file_path))
+        startupinfo = subprocess.STARTUPINFO()
+        startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
```

Comment:
This code doesn't work on Linux; you need to guard it so it only runs on Windows:
```suggestion
        if sys.platform == 'win32':
            startupinfo = subprocess.STARTUPINFO()
            startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
        else:
            startupinfo = None
```